packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue

```diff
@@ -11,18 +11,35 @@
 </template>
 
 <script setup lang="ts">
-import { ref, computed } from 'vue';
+import { watch, ref, computed } from 'vue';
 import Button from 'primevue/button';
 
 const props = defineProps<{
-	llmThought?: any;
+	llmThoughts: any[];
 }>();
 const showThoughts = ref(false);
-const thought = computed(() => props?.llmThought?.content?.thought ?? '');
+const thought = computed(() => {
+	let aString = '';
+	props.llmThoughts.forEach((ele) => {
+		const llmResponse = ele.content?.thought ?? ele.content?.text ?? '';
```

Comment:
Could Beaker makes sure this is sent to only one property? or those are completely different things?

Reply:
This is coming from https://github.com/jataware/beaker-kernel not https://github.com/DARPA-ASKEM/beaker-kernel
This is because one is an llm_thought object and the other is an llm_response object
I dont think its worth digging into this too much